sia-task-hunter/src/main/test/com/sia/hunter/StringHelperTest.java

```diff
@@ -0,0 +1,356 @@
+package com.sia.hunter;
+
+import com.sia.hunter.helper.StringHelper;
```

Comment:
Is this import needed? I'd think not if the package is `com.sia.hunter.helper` (as suggested elsewhere)